src/common/exception/GraknClientException.jl

```diff
@@ -2,13 +2,35 @@
 
 mutable struct GraknClientException <: Exception
     error_message::T where {T<:Union{<:AbstractGeneralError,<:Nothing}}
-    params
+    params::Optional{Tuple}
     individual_message::Union{Nothing, String}
-    cause::R where {R<:Exception}
+    cause::R where {R<:Union{Nothing, Exception}}
+end
+
+Base.show(io::IO, grakn_excption::GraknClientException) = Base.print(io,grakn_excption)
```

Comment:
naming:  leaving out just 1 letter in a long name to "abbreviate" it is just confusion and will cause mistakes.
`exc::TypeDBClientException` will work just fine.